pkg/types/proxy.go

```diff
@@ -22,7 +22,8 @@ import (
 )
 
 var (
-	ErrExit = errors.New("downstream process completed")
+	ErrExit   = errors.New("downstream process completed")
```

Comment:
it is easy to make mistakes when add new phase.
could you please optimize it ?
e.g.
```
var PhaseName = []string{
	InitPhase:  "InitPhase",
	DownFilter: "DownFilter",
	MatchRoute: "MatchRoute",
	End:        "End",
}
```

Reply:
done.